src/hydranfc/hydranfc_cmd_sniff.c

```diff
@@ -599,9 +599,11 @@ void sniff_write_eof_protocol(uint32_t timestamp_nb_cycles)
 	uint8_t data_buf[1];
 	uint8_t data;
 
-	data_buf[0] = RSSI_LEVELS;
-	Trf797xReadSingle(data_buf, 1);
-	data = data_buf[0];
+//	removed as it's too slow for sniffing
```

Comment:
Remove all comment lines related to RSSI for cleaner code and remove data_buf[1]; to avoid a warning